docs/source/contributing/developing.adoc

```diff
@@ -316,6 +316,13 @@ The following options are available:
 
 Tags::
 Use `tags` to ensure that scenarios and features containing at least one of the selected tags are executed.
+To select particular feature you can use its name as a tag.
+For example, `cmd-config.feature` contains `@cmd-config` tag through which the feature file can be selected and run: `make integration GODOG_OPTS=--tags=cmd-config`.
+There are also tags covering specific areas and usages of {project}:
+
+- The `command` tag indicates features which focus on testing of particular subcommands, for example `cmd-addons.feature` 
+- The `openshift` tag indicates features focusing on usage of OpenShift
+- The `addon` tag indicates features which test particular add-on functionality
```

Comment:
These are still not very clear explanations. I would suggest following
The `openshift` tag indicates that the feature file has tests on OpenShift.
The `addon` tag indicates feature file has tests particular to add-on functionality

Reply:
Ambiguous to say: "on OpenShift"... does this mean inside the platform, as a deployed application... does it test the application... or? "related to" would still be ambiguous, but indicates the target.
For addon, I am agreeing with Andreas. Also, if using 'file', be sure to use an article... like 'a feature file', or use a plural form to avoid.